.github/workflows/ci_submodule.yml

```diff
@@ -0,0 +1,72 @@
+name: CI for libsc check
+on:
+  pull_request:
+
+jobs:
+
+  linux-sub:
+    if: github.event.pull_request.draft == false
```

Comment:
Collaborators should be allowed to post a non-draft PR.

Reply:
Dode.